### DISCUSSION TITLE
VGM accel support

### DISCUSSION THREAD

**Discussion by jamisonderek** [rdefeo/pinball0#4] [Ideas]
any thought on adding VGM support for Flipper pinball? bumb it a little bit to help game play, but be careful because too much gives a tilt?

**Comment by rdefeo** [comment]
Ooh, interesting! I don't yet have a VGM, but maybe it can be in my future. I'm not sure if pinball is the best for motion control, but perhaps some other game ideas could benefit... Pinball0 won't be my last!

PS - great videos!

**Reply by jamisonderek** [reply]
Yeah, I guess using the Up button probably makes the most sense. The play feels strange when the VGM is attached (right button is a little too far left). Great job on the game!

![IMG_1902](https://github.com/user-attachments/assets/ddfa8427-2573-4336-a29f-a747a3fc788d)
